keyboards/k34/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "k34", 
+    "url": "https://github.com/wongjingping/k34", 
+    "maintainer": "qmk", 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"label":"E", "x":2, "y":0.0}, {"label":"W", "x":1, "y":0.25}, {"label":"R", "x":3, "y":0.25}, {"label":"Q", "x":0, "y":0.50}, {"label":"T", "x":4, "y":0.6}, {"label":"D", "x":2, "y":0.9}, {"label":"S", "x":1, "y":1.25}, {"label":"F", "x":3, "y":1.25}, {"label":"A", "x":0, "y":1.5}, {"label":"G", "x":4, "y":1.6}, {"label":"C", "x":2, "y":1.9}, {"label":"X", "x":1, "y":2.25}, {"label":"V", "x":3, "y":2.25}, {"label":"Z", "x":0, "y":2.5}, {"label":"B", "x":4, "y":2.6}, {"label":"Shift", "x":3, "y":3.25}, {"label":"Bksp", "x":4, "y":3.6}, {"label":"I", "x":2, "y":4.4}, {"label":"U", "x":1, "y":4.75}, {"label":"O", "x":3, "y":4.75}, {"label":"P", "x":4, "y":5.0}, {"label":"Y", "x":0, "y":5.1}, {"label":"K", "x":2, "y":5.4}, {"label":"J", "x":1, "y":5.75}, {"label":"L", "x":3, "y":5.75}, {"label":":", "x":4, "y":6.0}, {"label":"H", "x":0, "y":6.1}, {"label":"<", "x":2, "y":6.4}, {"label":"M", "x":1, "y":6.75}, {"label":">", "x":3, "y":6.75}, {"label":"?", "x":4, "y":7.0}, {"label":"N", "x":0, "y":7.1}, {"label":"fn", "x":1, "y":7.75}, {"label":"Space", "x":0, "y":8.1}]
```

Comment:
Still not correct. The two sides need to be next to each other, and the key ordering needs to match the layout macro. Use the Configurator to test this, press Ctrl+Shift+I and turn on fast input.

Reply:
Think I might have got it; was referencing some other similar keyboards but please let me know if I'm still missing something!